packages/react/src/themes/teams-dark/components/Menu/menuVariables.ts

```diff
@@ -1,9 +1,19 @@
-export interface MenuVariables {
-  color: string
-}
-
-export default (siteVars: any): MenuVariables => {
-  return {
-    color: siteVars.colors.white,
-  }
-}
+import { pxToRem } from '../../../../lib'
+import { MenuVariables } from '../../../teams/components/Menu/menuVariables'
+
+export default (siteVars: any): Partial<MenuVariables> => ({
+  borderColor: siteVars.black,
+
+  focusedBorder: `solid ${pxToRem(1)} ${siteVars.colors.white}`,
+  focusedBackgroundColor: 'transparent',
+
+  hoverBackgroundColor: siteVars.gray08,
+
+  activeColor: siteVars.colors.white,
+  activeBackgroundColor: siteVars.gray08,
+
+  verticalBackgroundColor: siteVars.gray10,
+
+  dividerHeight: pxToRem(2),
```

Comment:
Should divider height and border width really different between default and dark themes? Usually, it's just color differences.

Reply:
That's what I got from the redlines. Correct me if I'm wrong.